cnxpublishing/db.py

```diff
@@ -336,7 +334,7 @@ def _validate_derived_from(cursor, model):
     cursor.execute("""SELECT 't' FROM {} WHERE uuid::text = %s{}"""
                    .format(table, version_condition), args)
     try:
-        exists = cursor.fetchone()[0]
+        _exists = cursor.fetchone()[0]  # noqa
```

Comment:
I'm guessing you made these `_<name>` because it's an unused variable? Using just `_` is fine too and I believe flake8 is cool with it as a solution, because it says to the developer, "yes this thing is returning something, but we aren't using it."

Reply:
yup. I kept the name because it was documenting to some extent what was expected to be returned that was being ignored. I just tested, and flake8 2.6.2 still complained with '_', but I upgraded to 3.4.1, and now it's good with it. I'll convert to `_` (and maybe put info about what's being ignored in the trailing comment)

Reply:
Hmm, it looks like flake8v3 is a complete rewrite, which breaks a plugin I use, called flake8-putty. This allows me to have per directory config and error suppression, so that I can get diff line length warnings on testing/* than the body, on one flake8 call, so I think I'll leave this as is.